# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/properties-cpp/hotfix-cmake.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- properties-cpp-45863e849b39c4921d6553e6d27e267a96ac7d77/CMakeLists.txt.vanilla	2026-08-08 18:54:54.310671396 +0300
+++ properties-cpp-45863e849b39c4921d6553e6d27e267a96ac7d77/CMakeLists.txt	2026-08-08 18:55:10.146872740 +0300
--- properties-cpp-45863e849b39c4921d6553e6d27e267a96ac7d77/CMakeLists.txt.vanilla	2026-08-08 18:59:24.112621619 +0300
+++ properties-cpp-45863e849b39c4921d6553e6d27e267a96ac7d77/CMakeLists.txt	2026-08-08 19:02:19.871456996 +0300
@@ -4,8 +4,8 @@
 
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-include(cmake/EnableCoverageReport.cmake)
-include(cmake/PrePush.cmake)
+#include(cmake/EnableCoverageReport.cmake)
+#include(cmake/PrePush.cmake)
 include(GNUInstallDirs)
 
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror -Wall -pedantic -Wextra -fPIC -fvisibility=hidden -pthread")
@@ -40,6 +40,6 @@
 add_subdirectory(doc)
 add_subdirectory(data)
 add_subdirectory(include)
-add_subdirectory(tests)
+#add_subdirectory(tests)
 
 # enable_coverage_report(posix_process_test linux_process_test)

